spec/services/experiment_control_service_spec.rb

```diff
@@ -361,13 +283,13 @@
     it "only schedules reminders for PATIENTS_PER_DAY * number of days" do
       stub_const("ExperimentControlService::PATIENTS_PER_DAY", 1)
       patient1 = create(:patient, age: 80)
-      create(:encounter, patient: patient1, device_created_at: 100.days.ago)
+      create(:blood_pressure, patient: patient1, device_created_at: 100.days.ago)
       patient2 = create(:patient, age: 80)
-      create(:encounter, patient: patient2, device_created_at: 100.days.ago)
+      create(:blood_pressure, patient: patient2, device_created_at: 100.days.ago)
```

Comment:
All these test fixtures now rely on visits, which means an actual BP recorded or drug or RX, etc.  So we can't rely on creating encounters to setup our test data anymore.